frontend/src/metabase/home/components/EmbedHomepage/StaticTabContent.tsx

```diff
@@ -31,7 +31,7 @@ export const StaticTabContent = ({
           </List.Item>
         )}
         <List.Item>{jt`${
-          exampleDashboardId !== undefined ? t`Select` : `Create`
+          exampleDashboardId != null ? t`Select` : `Create`
```

Comment:
nit: Let's use `isNotNull` from `frontend/src/metabase/lib/types.ts`

Reply:
TIL isNotNull is using != and not !==

Reply:
For consistency I'll add it also later where I'm just doing `exampleDashboardId &&`